src/web/components/Content/Content.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Content from './';
+
+describe('Container', () => {
+  let content;
+  let wrapper;
+
+  beforeEach(() => {
+    content = <h1>Hello world</h1>;
+    wrapper = shallow(<Content>{content}</Content>);
+  });
+
+  it('should render children component', () => {
+    expect(wrapper.children()).toHaveLength(1);
+  });
+
+  it('should render a content without size by default', () => {
+    expect(wrapper.find('div.content').prop('className')).toBe('content');
+  });
+
+  it('should render a div with is-small class', () => {
```

Comment:
Your test message doesn't figure out that you need a size `props` to get that class.